gui/default/index.html

```diff
@@ -532,7 +532,9 @@ <h4 class="panel-title">
                       </tr>
                       <tr>
                         <th><span class="fas fa-fw fa-share-alt"></span>&nbsp;<span translate>Shared With</span></th>
-                        <td class="text-right" ng-attr-title="{{sharesFolder(folder)}}">{{sharesFolder(folder)}}</td>
+                        <td class="text-right">
+                          <span tooltip data-original-title="{{sharesFolder(folder)}} {{folderHasUnacceptedDevices(folder) ? '<br/>(<sup>1</sup>' + ('The remote device has not accepted sharing this folder.' | translate) + ')' : ''}}" ng-bind-html="sharesFolder(folder)" />
```

Comment:
```suggestion
                          <span tooltip data-original-title="{{sharesFolder(folder)}} {{folderHasUnacceptedDevices(folder) ? '<br/>(<sup>1</sup>' + ('The remote device has not accepted sharing this folder.' | translate) + ')' : ''}}" ng-bind-html="sharesFolder(folder)"></span>
```

Reply:
Only void elements like `input`, `br`, etc. can be self-closed in HTML. Otherwise, this results in invalid code which the browser interprets as `<span>` (without the slash, i.e. as if everything afterwards belonged to it) which may lead to unexpected consequences.

Reply:
Why?

Reply:
I'm just saying that this is invalid HTML. You are just not allowed to self-close non-void tags in HTML5, period.

Below is the explanation from https://validator.w3.org.

![image](https://user-images.githubusercontent.com/5626656/161385030-e597036a-5342-4c2e-925a-28d2a402ffe0.png)

Reply:
Sorry GitHub apparently didn't show your comment when I answered. That clears it up, thanks for educating me on the HTML details :-)